### PR TITLE
pin datashader >= 0.11

### PR DESCRIPTION
**Pull request by rabernat**
In https://github.com/holoviz/datashader/issues/598#issuecomment-660159877, @philippjfr recommended we force datashader >= 0.11. For some reason, 0.10 is still getting installed in our env, despite being 6 months old.

### CODE REVIEW

**Comment by scottyhq** [comment]
/condalock 

not sure either why datashader 0.10 is getting installed. could also try installing the conda environment locally with and without the pin to diagnose

File: pangeo-notebook/conda-linux-64.lock [hunk 1]
```diff
@@ -302,7 +301,7 @@ https://conda.anaconda.org/conda-forge/noarch/mercantile-1.1.5-pyh9f0ad1d_0.tar.
 https://conda.anaconda.org/conda-forge/noarch/munch-2.5.0-py_0.tar.bz2#31d9e9be500e25ff0050bc9f57a6bcd7
 https://conda.anaconda.org/conda-forge/linux-64/netcdf4-1.5.3-nompi_py37hec16513_103.tar.bz2#1274c6c94a897ee368fe64b1d5c41f98
 https://conda.anaconda.org/conda-forge/noarch/networkx-2.4-py_1.tar.bz2#5aaf85a2001a7d818148794f3099eb5e
-https://conda.anaconda.org/conda-forge/linux-64/numba-0.50.1-py37h0da4684_1.tar.bz2#e4896d6cdde22d25fa5cd48b31115074
+https://conda.anaconda.org/conda-forge/linux-64/numba-0.48.0-py37hb3f55d8_0.tar.bz2#2f5ebdbe82b91475079af2b431760a38
```
Comment (scottyhq) [inline pangeo-notebook/conda-linux-64.lock:304]: as far as i can tell the only package versions that dropped by pinning datashader>=0.11 are numba

File: pangeo-notebook/conda-linux-64.lock [hunk 2]
```diff
@@ -335,11 +334,11 @@ https://conda.anaconda.org/conda-forge/linux-64/pytest-5.4.3-py37hc8dfbb8_0.tar.
 https://conda.anaconda.org/conda-forge/linux-64/rasterio-1.1.5-py37hf4c872c_0.tar.bz2#576d5bb15396c47d73a3472c93d3b404
 https://conda.anaconda.org/conda-forge/linux-64/scikit-image-0.17.2-py37h0da4684_1.tar.bz2#37279ee84bc3eb6bcaae0752d0001da4
 https://conda.anaconda.org/conda-forge/linux-64/scikit-learn-0.23.1-py37h8a51577_0.tar.bz2#2874978913dbb836bb6abd4f982f55a0
-https://conda.anaconda.org/conda-forge/noarch/sparse-0.10.0-py_0.tar.bz2#c57a8ec66f6e74d038543df624327654
+https://conda.anaconda.org/conda-forge/noarch/sparse-0.9.1-py_0.tar.bz2#bfd3cd6640b1581ff64703a7048c487c
```
Comment (scottyhq) [inline pangeo-notebook/conda-linux-64.lock:337]: Also `sparse`

**Comment by rabernat** [comment]
According to @philippjfr:

> There's been some annoying regressions in numba 0.49 and 0.50 which may have prevented you from auto-upgrading because Datashader pinned older versions.

So this is consistent.

**Comment by scottyhq** [comment]
what's surprising is that numba isn't explicitly listed in our environment.yml, but `sparse` is: 
https://github.com/pangeo-data/pangeo-docker-images/blob/4e896bcb2d6bed31801d225c77785df2fcbd8190/pangeo-notebook/environment.yml#L62 

and `conda search sparse --info` shows this is where the incompatibility is coming from: 
```
sparse 0.10.0 py_0
------------------
file name   : sparse-0.10.0-py_0.tar.bz2
name        : sparse
version     : 0.10.0
build       : py_0
build number: 0
size        : 50 KB
license     : BSD-3-Clause
subdir      : noarch
url         : https://conda.anaconda.org/conda-forge/noarch/sparse-0.10.0-py_0.tar.bz2
md5         : c57a8ec66f6e74d038543df624327654
timestamp   : 2020-05-13 13:41:05 UTC
dependencies: 
  - numba >=0.49
  - numpy >=1.13
  - python >=3.5
  - scipy >=0.19
```

Seems like one of those things that will eventually work as new versions are released

**Pull request merged**